metron-interface/metron-rest/src/main/java/org/apache/metron/rest/service/impl/StormStatusServiceImpl.java

```diff
@@ -49,6 +44,11 @@ public StormStatusServiceImpl(Environment environment, RestTemplate restTemplate
   }
 
   @Override
+  public SupervisorSummary getSupervisorSummary(){
+    return restTemplate.getForObject("http://" + environment.getProperty(STORM_UI_SPRING_PROPERTY) + SUPERVISOR_SUMMARY_URL, SupervisorSummary.class);
```

Comment:
https://github.com/apache/metron/pull/602 changes how that works slightly.  Whichever one goes in second will need to adjust for the other one (which shouldn't be much work either way).  I don't have a preference on ordering, personally.